<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QPushButton" name="button_AmplifyBinaryImg">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="label_OrImg">
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>301</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>750</height>
       </size>
      </property>
      <property name="text">
       <string>原图</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="button_AmplifyOrImg">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>放大</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" colspan="4">
     <widget class="QLabel" name="label_BinaryImg">
      <property name="minimumSize">
       <size>
        <width>381</width>
        <height>301</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>750</height>
       </size>
      </property>
      <property name="text">
       <string>二值化图</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="button_LessenOrImg">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="slider_BinaryThre">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_SingleThre">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>单一阈值调节</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_ThresholdValue">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>127</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="button_lessenBinaryImg">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>15</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>缩小</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="button_ThreCon">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="start">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="openImg"/>
    <addaction name="saveImg"/>
    <addaction name="saveAsTXT"/>
   </widget>
   <widget class="QMenu" name="layer_Selected">
    <property name="title">
     <string>图层选择</string>
    </property>
    <widget class="QMenu" name="layer_M1">
     <property name="title">
      <string>M1</string>
     </property>
     <addaction name="single_Threshold"/>
     <addaction name="channel_Diff"/>
    </widget>
    <widget class="QMenu" name="layer_M2">
     <property name="title">
      <string>M2</string>
     </property>
     <addaction name="single_Threshold_2"/>
     <addaction name="channel_Diff_2"/>
    </widget>
    <addaction name="layer_M1"/>
    <addaction name="layer_M2"/>
    <addaction name="layer_Poly"/>
    <addaction name="layer_ST"/>
   </widget>
   <widget class="QMenu" name="img_Processing">
    <property name="title">
     <string>图片处理</string>
    </property>
    <widget class="QMenu" name="region_Selected">
     <property name="title">
      <string>区域选择</string>
     </property>
     <addaction name="cor_Input"/>
     <addaction name="local_Cover"/>
    </widget>
    <addaction name="channelSplit"/>
    <addaction name="region_Selected"/>
    <addaction name="layer_Overlay"/>
    <addaction name="img_Recover"/>
   </widget>
   <widget class="QMenu" name="setting">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <addaction name="start"/>
   <addaction name="img_Processing"/>
   <addaction name="layer_Selected"/>
   <addaction name="setting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openImg">
   <property name="text">
    <string>打开图片</string>
   </property>
  </action>
  <action name="saveImg">
   <property name="text">
    <string>保存图片</string>
   </property>
  </action>
  <action name="saveAsTXT">
   <property name="text">
    <string>导出TXT</string>
   </property>
  </action>
  <action name="layer_Poly">
   <property name="text">
    <string>Poly</string>
   </property>
  </action>
  <action name="layer_ST">
   <property name="text">
    <string>ST</string>
   </property>
  </action>
  <action name="single_Threshold">
   <property name="text">
    <string>单一阈值</string>
   </property>
  </action>
  <action name="channelSplit">
   <property name="text">
    <string>通道分离</string>
   </property>
  </action>
  <action name="single_Threshold_2">
   <property name="text">
    <string>单一阈值</string>
   </property>
  </action>
  <action name="channel_Diff">
   <property name="text">
    <string>通道差值</string>
   </property>
  </action>
  <action name="channel_Diff_2">
   <property name="text">
    <string>通道差值</string>
   </property>
  </action>
  <action name="img_Recover">
   <property name="text">
    <string>恢复原图</string>
   </property>
  </action>
  <action name="cor_Input">
   <property name="text">
    <string>坐标输入</string>
   </property>
  </action>
  <action name="local_Cover">
   <property name="text">
    <string>局部覆盖</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>自定义阈值</string>
   </property>
  </action>
  <action name="layer_Overlay">
   <property name="text">
    <string>图层叠加</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
